src/main/java/org/prebid/server/analytics/reporter/greenbids/model/CommonMessage.java

```diff
@@ -0,0 +1,39 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import org.prebid.server.auction.model.AuctionContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommonMessage {
+
+    public String version;
+    public String auctionId;
+    public String referrer;
+    public double sampling;
+    public String prebid;
+    public String greenbidsId;
+    public String pbuid;
+    public String billingId;
+    public List<AdUnit> adUnits;
+    public Long auctionElapsed;
+
+    public CommonMessage(
+            AuctionContext auctionContext,
+            GreenbidsConfig greenbidsConfig,
+            Long auctionElapsed,
+            String greenbidsId,
+            String billingId
+    ) {
```

Comment:
self-review: I did not added `ortb2Imp` field as on the side of Greenbids Analytics server it is marked as deprecated non standard field used to support custom GB analytics payload. Need think about whether to add it to the paylod

Reply:
fixed, I've added the ortb2Imp field to the payload [commit](https://github.com/prebid/prebid-server-java/pull/3096/commits/e64f3ff9ec41e149e0e8f39fdb39b4595c4c4cd1)

Here before the RTD adapter is implemented I've attributed `isExploration = true` to all adUnits and `keptInAuction = true` to all bidders as we do not filter them yet

Reply:
Also I've a question concerning UUIDs `tid` and `fingerprint`. I've been searching in PBJS RTD adapter, Analytics adapter and in data-processing but did not find where they are generated and added into ortb2Imp 🤔

Reply:
to revert